samza-core/src/main/java/org/apache/samza/config/TaskConfig.java

```diff
@@ -147,6 +147,9 @@ public class TaskConfig extends MapConfig {
       "task.transactional.state.retain.existing.state";
   private static final boolean DEFAULT_TRANSACTIONAL_STATE_RETAIN_EXISTING_STATE = true;
 
+  public static final String WATERMARK_IDLE_MS = "task.watermark.idle.ms";
```

Comment:
`WATERMARK_IDLE_TIMEOUT_MS`?

Reply:
Changed to WATERMARK_IDLE_TIMEOUT_MS. Thanks.